src/test/common/ios/plistBuddy.test.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {PlistBuddy} from "../../../common/ios/plistBuddy";
+
+import * as assert from "assert";
+import * as Q from "q";
+import * as sinon from "sinon";
+
+suite("plistBuddy", function() {
+    suite("commonContext", function() {
+        test("setPlistProperty should attempt to modify, then add, plist properties", function() {
+            const plistFileName = "testFile.plist";
+            const plistProperty = "myProperty";

```

Comment:
That's not inherent to the class though, that's just instance of how we use it at the moment